src/super_gradients/training/models/detection_models/yolo_base.py

```diff
@@ -437,27 +437,19 @@ def _check_strides_and_anchors(self):
 
         self.register_buffer('stride', m.stride)  # USED ONLY FOR CONVERSION
 
-    def _initialize_biases(self, cf=None):
+    def _initialize_biases(self):
         """initialize biases into Detect(), cf is class frequency"""
         detect_module = self._head._modules_list[-1]  # Detect() module
-        if isinstance(detect_module, Detect):
```

Comment:
Are you going to remove the `Detect` class definition? 
By analogy to this, please remove `if isinstance(m, Detect):` in `_check_strides_and_anchors()`. 
You can also get rid of `check_anchor_order(...)`